rd-net/RdFramework/RdId.cs

```diff
@@ -14,16 +14,21 @@ public struct RdId : IEquatable<RdId>
 
     private readonly long myValue;
 
-    public static RdId Define<T>(int? id = null)
+    public static RdId Define<T>(long? id = null)
     {
       return new RdId(id ?? Hash(typeof(T).Name));
     }
 
-    public static RdId Define(Type type, int? id = null)
+    public static RdId Define(Type type, long? id = null)
     {
       return new RdId(id ?? Hash(type.Name));
     }
 
+    public static RdId DefineByFqn(Type type)
```

Comment:
Please add comments that this method mustn't be used anyhow except C#-C# reflection interaction.